src/ploneintranet/workspace/browser/tiles/workspaces.py

```diff
@@ -41,7 +41,7 @@ def my_workspaces(context):
         path=ws_path,
     )
     workspaces = []
-    for brain in brains:
+    for brain in brains[:5]:
         css_class = escape_id_to_class(brain.getId)
```

Comment:
Isn't that box only meant to give a quicklink style entry to the recent ones? If I don't find it there I can still go to workspaces and find it there. A filter by "my workspaces" may be an option there.

And if not, what about our managerial colleagues who can see all workspaces? They will have a list with 100 items in there? (Asking because I've been there)